libs/labelbox/tests/data/serialization/ndjson/test_generic_data_row_data.py

```diff
@@ -0,0 +1,79 @@
+from labelbox.data.annotation_types.data.generic_data_row_data import (
```

Comment:
I think testing withevery annotation was a bit overkill and would be complicated to add in without deserialization existing. I would have to rewrite that `annotations_by_media_type` fixture to provide Pydantic models over just the JSON and that fixture is pretty large. I wanted to simplify it by just seeing if the GenericDataRow serializes correctly. We also test for the GenericDataRow type with just the dictionary data={"global_key":global_key} and data={"uid":data_row_id} in our end to end testing

Reply:
Id have to rewrite most of this file https://github.com/Labelbox/labelbox-python/blob/develop/libs/labelbox/tests/data/annotation_import/conftest.py to make it work :disappointed: specifically the interfaces. Since we cant deserialize the JSON anymore directly to pydantic objects.